Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -375,7 +375,20 @@ def replace_featured_field(self, field_type: str, fields: list[dict]) -> dict:
 
         return reply.get('reply')
 
+    def get_tenant_info(self):
+        request_data: Dict[str, Any] = {}
+        reply = self._http_request(
+            method='POST',
+            url_suffix='/system/get_tenant_info/',
+            json_data={'request_data': request_data},
```

Comment:
no need to create request_data if its empty dict, remove it and change line 383 to:

```
            json_data={'request_data': request_data},
```